src/utils/test-data.js

```diff
@@ -0,0 +1,53 @@
+const firstNames = [
```

Comment:
in real project I would suggest to use libraries like fakerjs if there is no special requirements

just an advice